config_parser_test.cc

```diff
@@ -1,11 +1,71 @@
+#include <sstream>
+#include <string>
 #include "gtest/gtest.h"
 #include "config_parser.h"
 
-TEST(NginxConfigParserTest, SimpleConfig) {
-  NginxConfigParser parser;
-  NginxConfig out_config;
 
-  bool success = parser.Parse("example_config", &out_config);
+// Test fixture for testing parsing a string literal
+class NginxParseStringConfigTest : public ::testing::Test {
```

Comment:
Consider merging this test fixture with the other one for better organization and less repeated code.